frontend/src/utils/index.js

```diff
@@ -728,7 +737,15 @@ export function selectedImageIsNotLatest (machineImage, machineImages) {
 export function k8sVersionExpirationForShoot (shootK8sVersion, shootCloudProfileName, k8sAutoPatch) {
   const allVersions = store.getters.kubernetesVersions(shootCloudProfileName)
   const version = find(allVersions, { version: shootK8sVersion })
-  if (!version || !version.expirationDate) {
+  if (!version) {
+    return {
+      version: shootK8sVersion,
+      expirationDate: 'Unknown',
```

Comment:
is this only used for display purposes?

Reply:
You are right, I'm also not happy with the 'Unknown' String - basically this is a hack, as the expirationDate string must not be undefined in order to make the logic work. Basically the correct value would be a timestamp in the past, so I guess we could also generate an expired timestamp

Reply:
I guess this is now done, right?

Reply:
changed it to UNKNOWN_EXPIRED_TIMESTAMP = '1970-01-01T00:00:00Z'
I think it is resolved now